lib/models/actor.js

```diff
@@ -1,15 +1,24 @@
 const mongoose = require('mongoose');
 const { Schema } = mongoose;
+const { ObjectId } = Schema.Types;
 
 const schema = new Schema({
   name: {
     type: String,
     required: true
   },
-  birthYear: {
-    type: Number,
+  dob: {
+    type: String,
+    required: true
+  },
+  pob: {
+    type: String,
     required: false
-  }
+  },
+  films: [{
```

Comment:
you don't have to put films into the actor model - you can just add it in the get by id route